content/docs/0.19.x/manage/plan/index.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Plan your project
+description: Create and delete a project in Keptn.
+weight: 10
+keywords: [0.16.x-manage]
+aliases:
+---
+
+It is useful to think about what you want your Keptn project to do
+before you begin implementing it.
+This page summarizes some things you should consider.
+You can then use the [Create project](../project) page
+for instructions about implementing a project.
+
+It is important to understand that a Keptn project is **not** a pipeline.
+Instead, activities are triggered by specific events.
+For a broader explanation of this, you can read the seminal
+[Continuous Delivery without pipelines -- How it works and why you need it](https://medium.com/keptn/continuous-delivery-without-pipelines-7e84db8c8261) article.
+
+## What should this project do?
+
+Before you can run the Keptn command to create a project,
+you must populate a [shipyard.yaml](../../reference/files/shipyard) file
+that defines the `stages` for your project.
+At the current time, it is not possible to add or remove stages
+after you create your project.
+
+A stage is a grouping of activities to be executed and it consistes of one or more sequences that define the actual tasks to perform. The shipyard.yaml file you use to create a project must show the stages to be used – at least one stage is required but most projects have multiple stages – and each stage must have a name property defined. For example,:
+
+```
+apiVersion: spec.keptn.sh/0.2.3
+kind: "Shipyard"
+metadata:
+  name: "shipyard-sockshop"
+spec:
+  stages:
+    - name: "dev"
+    - name: "hardening"
+    - name: "production"
+```
+
+In order to do any work, each stage must have at least one sequence and most sequences will contain one or more tasks but this information can be added to the shipyard after you create your project.
```

Comment:
you say here that a stage always needs to have at least one sequence - but - in the example above there are no sequences. I think its still true that there is always the "evaluation" sequence there by default (=implicit) even if you dont specify it. this is why a basic shipyard that only contain stages actually works as every stage by default has a sequence called "evaluation" with a single task that is also called evaluation. A user can however define his own evaluation sequence with any set of tasks

Reply:
Should we be “advertising” this though? I understand why it is so, but It breaks the premise that the Shipyard is the source of truth and IMO promotes bad habits.

Reply:
I stole the statement that "Each stage must have  a sequence" from somewhere in the existing docs and then started to see that this didn't exactly make sense.  Thanks for the explanation, Andi -- now I understand.   For this section, I'm not sure we even need to mention sequences.  But I am going to add this subtle detail to the shipyard reference page.

Reply:
This is way too in-the-weeds for this document but I added info about the implicit evaluation sequence to the shipyard reference file in a separate PR